src/Employer/Employer.Web/Views/Part1/Training/Training.cshtml

```diff
@@ -41,7 +37,8 @@
 
                 <div esfa-validation-marker-for="StartDate" class="form-group">
                     <fieldset>
-                        <legend class="form-label-bold">Possible start date<span class="visuallyhidden"> (in numeric format DD MM YYYY)</span></legend>
+                        <legend class="form-label-bold">When do you expect the apprenticeship to start?<span class="visuallyhidden"> (in numeric format DD MM YYYY)</span></legend>
+                        <span class="form-hint">For example, 30 3 2019</span>

```

Comment:
Can we use the example dates as we did in the example vacancy instead of hard coding the date.

Reply:
👍 Good idea. I'll check if this is OK